src/renderer/components/composer/DisabledMessageInput.tsx

```diff
@@ -7,40 +7,39 @@ type Props = {
   reason?: DisabledChatReasons
 }
 
-const DisabledMessageInput = forwardRef<any, Props>(
-  ({ reason }: Props, ref) => {
-    const tx = useTranslationFunction()
-
-    const reasonMessage = useMemo(() => {
-      switch (reason) {
-        case DisabledChatReasons.MAILING_LIST:
-          return tx('messaging_disabled_mailing_list')
-        case DisabledChatReasons.NOT_IN_GROUP:
-          return tx('messaging_disabled_not_in_group')
-        case DisabledChatReasons.DEADDROP:
-          return tx('messaging_disabled_deaddrop')
-        case DisabledChatReasons.UNKNOWN:
-        default:
-          return tx('messaging_disabled_unknown')
-      }
-    }, [reason, tx])
-
-    // If no reason was given we return no component
-    if (!reason) {
-      return null
+const DisabledMessageInput = ({ reason }: Props) => {
+  const tx = useTranslationFunction()
+
+  const reasonMessage = useMemo(() => {
+    switch (reason) {
+      case DisabledChatReasons.MAILING_LIST:
+        return tx('messaging_disabled_mailing_list')
+      case DisabledChatReasons.NOT_IN_GROUP:
+        return tx('messaging_disabled_not_in_group')
+      case DisabledChatReasons.DEADDROP:
+        return tx('messaging_disabled_deaddrop')
+      case DisabledChatReasons.DEVICE_CHAT:
+        return tx('messaging_disabled_device_chat')
+      default:
```

Comment:
```suggestion
      case DisabledChatReasons.UNKNOWN:
        return tx('messaging_disabled_unknown');
      default:
```

Reply:
Ah, sorry, I've misread your point above.

We do not have any message for `messaging_disabled_unknown`, can I just go ahead and add one for `en`?

Reply:
we also put an untranslated string there, point is just that people should not get a crash screen in normal usage and english error is better than that.

Reply:
we developers on the other hand should get a crash screen when we mess up in this case that's why I proposed it that way ;)

Reply:
> we also put an untranslated string there

I receive an annoying warning in my console that the translation is missing, this is why I've created that PR actually first-hand 😅